### PR TITLE
withErrorBoundary HOC

### PR DESCRIPTION
**Pull request by JoshuaRogan**
* Create a new error boundary component. Currently used in Fandom creator
* Added an event logger function 

@Wikia/cake

### CODE REVIEW

File: source/utils/config.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+// true if hosted on fandom.com
+const IS_PRODUCTION_APP = process.env.env === 'production';
+
+const SERVICES_BASE_URL = isFandomCom() ? 'https://services.fandom.com/' : 'https://services.fandom-dev.us/';
+
+const config = {
+    isFandomCom,
+    IS_PRODUCTION_APP,
```
Comment (mklucsarits) [inline source/utils/config.js]: I'd prefer to use consistent casing for each of these property names.

**Comment by mklucsarits** [comment]
Seems OK to me but would appreciate a second set of eyes.

**Comment by JoshuaRogan** [comment]
I don't have access to those travis builds. Are they running in our main travis account?

**Comment by vforge** [comment]
Talked offline, let's go!

**Pull request merged**